### PR TITLE
storing token in new table meican_cilogon_auth

### PR DESCRIPTION
**Pull request by mahidhar1g**

### CODE REVIEW

File: modules/circuits/controllers/NodesController.php [hunk 1]
```diff
@@ -70,14 +73,138 @@ public function actionCreate(){ // this route manages the view and backend logic
     }
     
     public function actionShow() {   // this function manages the mapping of SDX-topology and displays on the MEICAN UI
-   
+      
+      $api_url=API_URL;
+      $meican_url=MEICAN_URL;
+
+      $userId = Yii::$app->user->id;
```
Comment (usmanazFIU) [inline modules/circuits/controllers/NodesController.php:80]: add comments

File: modules/circuits/controllers/NodesController.php [hunk 2]
```diff
@@ -70,14 +73,138 @@ public function actionCreate(){ // this route manages the view and backend logic
...
+      $rows = (new \yii\db\Query())
+          ->select(['token', 'expiration'])
+          ->from('meican_cilogon_auth')
+          ->where(['user_id' => $userId])
+          ->all();
+      
+      date_default_timezone_set('America/New_York');
```
Comment (usmanazFIU) [inline modules/circuits/controllers/NodesController.php:87]: add comments regarding how the token is updating/inserting and how the expiration is working.

**Review by usmanazFIU — APPROVED** [review]

**Comment by usmanazFIU** [comment]
nice work

File: modules/circuits/controllers/NodesController.php (hunk 2 above)
Comment (mahidhar1g) [reply modules/circuits/controllers/NodesController.php:88]: Hi Usman, I have added the comments and pushed the code.

**Pull request merged**